### PR TITLE
Less invasive random implementation

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

**Comment by coreyfloyd** [comment]
@joewalsh also here… sorry

**Comment by joewalsh** [comment]
@coreyfloyd updated

File: Wikipedia/Code/WMFRotationRespectingNavigationController.m [hunk 1]
```diff
@@ -3,29 +3,32 @@
...
-    UIViewController* vcToRespect = self.presentedViewController ? self.presentedViewController : self.topViewController;
-    if (vcToRespect && [vcToRespect isKindOfClass:[UIAlertController class]]) {
-        return [vcToRespect shouldAutorotate];
+    if (self.presentedViewController && ![self.presentedViewController isKindOfClass:[UIAlertController class]]) {
+        return self.presentedViewController.shouldAutorotate;
+    } else if (self.topViewController) {
+        return self.topViewController.shouldAutorotate;
     } else {
-        return [super shouldAutorotate];
+        return YES;
```
Comment (montehurd) [inline Wikipedia/Code/WMFRotationRespectingNavigationController.m]: haha oops! that could be it :) testing...

Comment (coreyfloyd) [inline Wikipedia/Code/WMFRotationRespectingNavigationController.m]: Should the default be YES or NO?

Comment (joewalsh) [inline Wikipedia/Code/WMFRotationRespectingNavigationController.m]: YES

Comment (joewalsh) [inline Wikipedia/Code/WMFRotationRespectingNavigationController.m]: (just a guess)

**Comment by montehurd** [comment]
iOS 8 dice animation has issues. good for merge anyway since we mostly want to start testing the interaction. can address dice later
👍

**Pull request merged**